src/sparse/triplet_iter.rs

```diff
@@ -108,23 +108,26 @@ where
     DI: Clone + Iterator<Item = &'a N>,
 {
     /// Consume `TriMatIter` and produce a CSC matrix
-    pub fn into_csc(self) -> CsMatI<N, I>
+    pub fn into_csc<Iptr: SpIndex>(self) -> CsMatI<N, I, Iptr>
```

Comment:
> There might be a way to let Iptr = I as default?

Unfortunately I don't think it's enabled on functions, see https://github.com/rust-lang/rust/issues/36887 (looks like there is some debate though). For the time being I think we can accept the occasional need for type annotations, especially since the type inference might actually help in functions larger than the tests.